src/DocumentationExamples/CodeExamples/ShouldMatchApprovedExamples.ApprovedFileDoesNotExist.exceptionText.approved.txt

```diff
@@ -1,6 +1,6 @@
 To approve the changes run this command:
-copy /Y "C:\PathToCode\shouldly\src\DocumentationExamples\ShouldMatchApprovedExamples.ApprovedFileDoesNotExist.received.txt" "C:\PathToCode\shouldly\src\DocumentationExamples\ShouldMatchApprovedExamples.ApprovedFileDoesNotExist.approved.txt"
+copy /Y "C:\Users\OsamaLone\source\repos\shouldlyosama\src\DocumentationExamples\ShouldMatchApprovedExamples.ApprovedFileDoesNotExist.received.txt" "C:\Users\OsamaLone\source\repos\shouldlyosama\src\DocumentationExamples\ShouldMatchApprovedExamples.ApprovedFileDoesNotExist.approved.txt"
```

Comment:
These files should be reverted as they contain paths from your environment

Reply:
i tried doing it but for some reason they were not reverted. The thing is i dont have time as i am currently busy with my exams. But when i do get it i ll update them or you can replace these by pulling this pull request or by going to my [repository](https://github.com/loneosama/shouldly) to pull it. Or when i get time i ll make changes in your repository if needed and you can send the pull request.
Thanks